### ISSUE TITLE
Custom domain is deleted from settings

### ISSUE THREAD

**Issue by yangwao** [L33T-KR3W/push-dir#26]
I just wondering, every time when I use push-dir, `npm run deploy`
```
"deploy": "push-dir --dir=dist --branch=master --cleanup",
```
I just get my custom domain removed from settings of repository.
Any tips why is this happening?

**Comment by scott113341** [comment]
Hi @yangwao, I don't have time right now to verify, but my guess is that this is happening because you have your `dist` folder added to your `.gitignore`

https://github.com/yangwao/yangwao.github.io/blob/dev/.gitignore

**Comment by yangwao** [comment]
yes, because Nuxt will generate static content to /dist folder, which will `push-dir` push to the master branch. 

Is there any other way how to accomplish this or is there some problems using /dist for this scenario?

Thanks, push-dir is great module! :)

**Comment by scott113341** [comment]
I think the only way to achieve this with the current approach is to remove `dist` from the `.gitignore` and then do something like this:

```
$ npm run build
$ git add . && git commit -m build
$ npm run deploy
```

**Comment by mmoscosa** [comment]
Am having the same prpblem, 

I have removed the dist folder from gitignore with no luck; you can see in the video below.

https://www.useloom.com/share/a96acdbaff6c4ca0aaea4a97534aa129

**Comment by yangwao** [comment]
@mmoscosa yes, looks pretty same. I think Nuxt should not affect this. I'm using Nuxt too.
I already moved to netlify for now, but I'd like to use github pages too for some projects with push-dir.

**Comment by masonhipp** [comment]
Also having the same problem with push-dir and Nuxt as static generator. I also removed the dist directory from my gitignore file and that didn't fix it.

**Comment by scott113341** [comment]
https://github.com/nuxt/docs/blob/master/en/faq/github-pages.md

**Comment by yipcma** [comment]
You need to put the CNAME in static

**Comment by remino** [comment]
Although no longer documented by GitHub, using a CNAME file still works. I was having the same problem, and adding a single line with my custom domain name in `static/CNAME`, as @yipcma suggested, then running `push-dir` worked.

**Comment by ihorbond** [comment]
^ this works !
